salt/metalk8s/addons/prometheus-operator/deployed/service-configuration.sls

```diff
@@ -39,9 +39,16 @@ Create grafana-config ConfigMap:
           config.yaml: |-
             apiVersion: addons.metalk8s.scality.com
             kind: GrafanaConfig
-            spec:
-              deployment:
-                replicas: 1
+            spec: {}
+        # example configuration
+          # example.yaml: |-
+          #   apiVersion: addons.metalk8s.scality.com
+          #   kind: GrafanaConfig
+          #   spec:
+          #     # Configure the Grafana Deployment
+          #     deployment:
+          #       replicas: 1
```

Comment:
Not sure we want to keep this here as it does not really help, we should rather have some documentation exposing the CSC defaults.

Reply:
Ok, going to open a documentation ticket to track this